packages/react-components/src/components/tooltip/tooltip.css

```diff
@@ -0,0 +1,51 @@
+.tooltip {
```

Comment:
this is used in multiple placess, especially the resource explorer, make sure this hasnt broken those